tests/conftest.py

```diff
@@ -37,26 +37,15 @@
     pass
 
 from mava import specs as mava_specs
-from mava.environment_loop import ParallelEnvironmentLoop, SequentialEnvironmentLoop
+from mava.environment_loop import ParallelEnvironmentLoop
 from mava.types import Observation, Reward
-
-try:
-    from mava.utils.environments.open_spiel_utils import load_open_spiel_env
-    from mava.wrappers.open_spiel import OpenSpielSequentialWrapper
-except ImportError:
-    pass
 from mava.utils.wrapper_utils import convert_np_type
 from mava.wrappers.pettingzoo import (
     PettingZooAECEnvWrapper,
     PettingZooParallelEnvWrapper,
 )
-from tests.enums import EnvSource, EnvSpec, EnvType, MockedEnvironments
-from tests.mocks import (
-    ParallelMAContinuousEnvironment,
-    ParallelMADiscreteEnvironment,
-    SequentialMAContinuousEnvironment,
-    SequentialMADiscreteEnvironment,
-)
+from tests.enums import EnvSource, EnvSpec, MockedEnvironments
+from tests.mocks import ParallelMAContinuousEnvironment, ParallelMADiscreteEnvironment
```

Comment:
Can we please remove the "MA" from all the classes. Feels unnecessary to specify that the env is multi-agent in a multi-agent library.

Reply:
I tried to do this. But at some places they are necessary, because we import a ParallelContinuousEnvironment and wrap it to make it multi-agent.

Reply:
Ah I see, ok no problem then, makes sense and happy to keep it as is then. 👍